src/proposals/20250626/SparkEthereum_20250626.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.25;
+
+import { IMetaMorpho, MarketParams } from 'metamorpho/interfaces/IMetaMorpho.sol';
+
+import { Ethereum } from 'spark-address-registry/Ethereum.sol';
+
+import { SparkPayloadEthereum } from "../../SparkPayloadEthereum.sol";
+
+/**
+ * @title  June 26, 2025 Spark Ethereum Proposal
+ * @notice Spark DAI Morpho Vault:
+ *         - Onboard PT-syrupUSDC-28Aug2025/DAI
+ *         - Onboard PT-USDe-25Sept2025/DAI
+ * @author Phoenix Labs
+ * Forum:  https://forum.sky.money/t/june-26-2025-proposed-changes-to-spark-for-upcoming-spell/26663
+ * Vote:   
```

Comment:
Links:
https://vote.sky.money/polling/QmXKUVXi
https://vote.sky.money/polling/QmcGPTMX
https://vote.sky.money/polling/QmWtGgPH

If you ever want to find them just go here https://vote.sky.money/polling?category=spark

Reply:
added votes